code/stable-website/index.html

```diff
@@ -138,6 +138,12 @@ <h1>About</h1>
         <p class="paragraph">
           A list of papers and documents written about Drasil can be found in the <a href="https://github.com/JacquesCarette/Drasil/wiki/Drasil-Papers-and-Documents">Drasil Papers and Documents</a> wiki page
```

Comment:
Alternatively, you can also append to this paragraph (fixing its missing period too :smile: ) and inspiration from the `README.md` file, perhaps with something along the lines of:
```
In particular, there is a [position paper](link) outlining our original ideas, a [poster](link), and a [recent paper](link) discussing key concepts.
```